### PR TITLE
docs(many): function based examples are added to ToggleButton, Trucat…

### PR DESCRIPTION
**Pull request by ToMESSKa**
…eText and TimeSelect

Closes: INSTUI-4153

### CODE REVIEW

**Review by balzss — COMMENTED** [review]
looks good, all problems are from the original examples but we should fix those regardless

File: packages/ui-buttons/src/ToggleButton/README.md [hunk 1]
```diff
@@ -31,53 +55,86 @@ class Example extends React.Component {
...
+        </View>
+      )
+    }
   }
+  render(<InverseExample />)
+  ```
+
+- ```js
+  const InverseExample = () => {
+    const [lockedStatus, setLockedStatus] = useState('off')
```
[the remark this answers is not in the document]
Comment (matyasf) [reply packages/ui-buttons/src/ToggleButton/README.md:110]: agree

File: packages/ui-time-select/src/TimeSelect/README.md [hunk 2]
```diff
@@ -24,36 +24,54 @@ type: example
...
-type: example
----
-class Example extends React.Component {
-  state = {
-    value: '2020-05-18T23:59:00'
-  }
+- ```javascript
+  class Example extends React.Component {
+    state = {
+      value: '2020-05-18T23:59:00'
```
[the remark this answers is not in the document]
Comment (matyasf) [reply packages/ui-time-select/src/TimeSelect/README.md:30]: I remember that this use case was specifically requested (default value could be something that is not in the dropdown), so I'd leave it here. The explanation when this is used to set a deadline its default is 1 min before midnight, which the user should be able to change.

When we make a new `TimeSelect` it should handle this better.

**Review by matyasf — APPROVED** [review]

File: packages/ui-buttons/src/ToggleButton/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ToMESSKa) [reply packages/ui-buttons/src/ToggleButton/README.md]: I simplified the code with a boolean

File: packages/ui-buttons/src/ToggleButton/README.md [hunk 3]
```diff
@@ -31,53 +55,86 @@ class Example extends React.Component {
...
+      >
         <ToggleButton
           status={locked ? 'pressed' : 'unpressed'}
           color="primary-inverse"
           renderIcon={locked ? IconLockSolid : IconUnlockLine}
           screenReaderLabel={locked ? 'Unlock assignment' : 'Lock assignment'}
           renderTooltipContent={locked ? 'Unlock' : 'Lock'}
-          onClick={this.handleLocked}
+          onClick={handleLocked}
           mountNode={() => document.getElementById('inverseToggleContainer')}
```
[the remark this answers is not in the document]
Comment (ToMESSKa) [reply packages/ui-buttons/src/ToggleButton/README.md]: I works without it, removed

**Review by balzss — APPROVED** [review]

**Pull request merged**